enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/BoltInteraction.java

```diff
@@ -68,11 +68,12 @@
 
     EnterpriseAuthManager authManager;
 
-    public BoltInteraction( Neo4jWithSocket server ) throws IOException
+    public BoltInteraction( Neo4jWithSocket server ) throws Throwable
     {
         this.server = server;
         GraphDatabaseFacade db = (GraphDatabaseFacade) server.graphDatabaseService();
         authManager = db.getDependencyResolver().resolveDependency( EnterpriseAuthManager.class );
+        authManager.start();
```

Comment:
That was to ensure that the integration test environment has the same initial default settings as the server (and as the user management command-line).